### DISCUSSION TITLE
False positives with fly.io shared IPv4 (Anti-Maleware List)

### DISCUSSION THREAD

**Discussion by claudiiii** [DandelionSprout/adfilt#1007] [General]
Hi, 

I got confused why my domain ended up blocked by the "Dandelion Sprout's Anti-Malware List".
In the [latest commit](https://github.com/DandelionSprout/adfilt/commit/2b0705861bde02511e1fdd72edf470b2811ecb8c) of that list, a bunch of domains got added which all point to the same IPv4 address.

Here's a small excerpt
```
> dig smith.cm A +short
66.241.124.36
> dig thebrianemory.com A +short
66.241.124.36
> dig usepigeons.com A +short
66.241.124.36
```

This is a shared IPv4 address by https://fly.io. They introduced them [earlier this year](https://community.fly.io/t/announcement-shared-anycast-ipv4/9384) and using a dedicated IPv4 address costs extra.

I think it would be better do not block those domains just for pointing to the same IPv4 address. Maybe we can find a solution?

**Comment by DandelionSprout** [comment]
Thanks for this tip. I was doing some research on very questionable GitHub typo squat addresses, and came across `66.241.124.36`, whose list of domains looked a bit odd. Now that I know several of those domains are intended for legitimate use, I'll handle it sometime tonight, hopefilly within a couple hours.

**Comment by claudiiii** [comment]
Thanks for the quick fix!
